tests/cmdline/groups/test_dynamic.py

```diff
@@ -17,7 +17,7 @@ class Model(BaseModel):
         union_type: t.Union[int, float] = Field(title='Union type')
         without_default: str = Field(title='Without default')
         with_default: str = Field(title='With default', default='default')
-        with_default_factory: str = Field(title='With default factory', default_factory=lambda: True)
+        with_default_factory: str = Field(title='With default factory', default_factory=lambda: True)  # type: ignore[assignment]
```

Comment:
It is test, so I just ignore it.

Reply:
Actually, when I run mypy 1.13 on this, the `# type: ignore[assignment]` is not necessary?